lib/utils.js

```diff
@@ -252,9 +254,9 @@ exports.files = function (dir, ext, ret) {
     .filter(ignored)
     .forEach(function (path) {
       path = join(dir, path);
-      if (statSync(path).isDirectory()) {
+      if (lstatSync(path).isDirectory()) {
         exports.files(path, ext, ret);
-      } else if (path.match(re)) {
+      } else if (path.match(re) && existsSync(path)) {
```

Comment:
Using `existsSync` here still silently ignores the problem rather than making the error message useful as discussed in the previous review, doesn't it?